assets/scripts/lobby/buttonEventListeners2.js

```diff
@@ -224,10 +224,6 @@ $('#createNewRoomButton').on('click', (data) => {
   }
 });
 
-$('#joinQueueButton').on('click', () => {
-  socket.emit('join-queue');
+$('#queueButton').on('click', () => {
+  socket.emit('queue-request');
```

Comment:
This should have an if statement right? 

If joinedQueue then emit leave-queue. 
else emit join-queue